partner_risk_insurance/models/res_partner.py

```diff
@@ -0,0 +1,58 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _inverse_oca_credit_limit(self):
+        """The credit_limit field already exists in account, we use an inverse to define
+        the value without altering the standard operation."""
+        for item in self:
+            if "use_partner_credit_limit" not in self._fields or (
+                "use_partner_credit_limit" in self._fields
+                and not item.use_partner_credit_limit
+            ):
+                item.credit_limit = (
+                    item.company_credit_limit + item.insurance_credit_limit
+                )
+
+    credit_limit = fields.Float(store=True)
+    company_credit_limit = fields.Float(
+        "Company's Credit Limit",
+        help="Credit limit granted by the company.",
+        tracking=True,
+        inverse="_inverse_oca_credit_limit",
+    )
+    insurance_credit_limit = fields.Float(
+        "Insurance's Credit Limit",
+        help="Credit limit granted by the insurance company.",
+        tracking=True,
+        inverse="_inverse_oca_credit_limit",
+    )
```

Comment:
Why not remove this field and let only the `credit_limit`? Is any difference between `insurance_credit_limit` and `credit_limit`?
On the view, just use the same field with a different string.

Reply:
Ok, I understand what you are trying to do, but there is also a small detail, in v15 the field was added by `base`, and in v16 the field (`credit_limit`) is added by `account`.

Taking into account that this module does not depend on `account` because `account_credit_control` already exists, what would be the process to follow in this case (maybe in this module leave the current behavior of compute, and in `account_credit_control` make it compatible with company_dependent as you indicate).

Reply:
`account_credit_control` module is for other things. I think this one can depend on `account` directly.